File: src/main/java/edu/dcccd/trans/entity/Role.java
package edu.dcccd.trans.entity;

import lombok.*;
import javax.persistence.*;

@Entity
@Table(name = "roles")
@Data
@NoArgsConstructor
public class Role {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String role;
    @Column(name = "user_id")
    private Long userId;

    public Role(String role, Long userId) {
        this.role = role;
        this.userId = userId;
    }
}

File: src/main/kotlin/edu/dcccd/trans/entity/Role.kt
package edu.dcccd.trans.entity

import javax.persistence.*

@Entity
@Table(name = "roles")
data class Role(@Id @GeneratedValue(strategy = GenerationType.IDENTITY) val id: Long?,
                val role: String,
                @Column(name = "user_id") val userId: Long?
) {
    constructor() : this(null, "", null)
}